patches/Terraria/Terraria/Terraria.csproj.patch

```diff
@@ -74,13 +68,13 @@
 +    <None Remove="Libraries/Windows/**" />
 +    <None Remove="Libraries/XNA/**" />
 +    <None Remove="Libraries/net40/**" />
-+    <None Update="Libraries/Native/**" CopyToOutputDirectory="PreserveNewest" />
++    <Content Include="Libraries/Native/**" CopyToOutputDirectory="PreserveNewest" />
 +  </ItemGroup>
 +  <Target Name="CopyToSteamDir" AfterTargets="Build">
 +    <ItemGroup>
 +      <BinFiles Include="$(TargetDir)**" />
 +    </ItemGroup>
 +    <Message Text="Copying $(AssemblyName) to Steam Dir..." Importance="high" />
-+    <Copy SourceFiles="@(BinFiles)" DestinationFolder="$(TerrariaSteamPath)/%(RecursiveDir)" />
++    <Copy SourceFiles="@(BinFiles)" DestinationFolder="$(tModLoaderSteamPath)/%(RecursiveDir)" SkipUnchangedFiles="True" />
```

Comment:
Intentional change here? We previously built the Terraria sln into the Terraria steam dir. I don't think Terraria.csproj has the required asset path redirects for the side-by-side content sharing etc.

Reply:
Fair point I didn't wanna copy the full block there, I should probably just point and do all the stuff tML does, because now a build that is not `Debug` will be called `Terraria.exe` overwriting the game, also a bunch of modified dlls might be used by vanilla terraria due to name clashing (mainly ReLogic).